circle.yml

```diff
@@ -332,7 +332,7 @@ workflows:
           context: ocim-frontend-stage
           requires:
             - frontend-test
-      # Frontend deployment (only on master branch)
+      # Frontend deployment (only on master and stage branches)
       - frontend-deploy:
           name: frontend-deploy-stage
```

Comment:
Deploy stage should only happen in the `stage` branch, not on `master` too. Otherwise merges to `master` will overwrite changes in `stage`

Reply:
@giovannicimolin For the first time I had the same opinion, though in case we deploy backend from master to staging automatically and deploy frontend by merging to staging branch, that would make really hard to follow what is happening in the staging environment. 

In case we deploy completely separately (backend automatically by merging to master, frontend manually by merging to stage) we could encounter with the same issue what we have right now, but for the other direction, or am I missed something?

Reply:
I overlooked this, since backend is not deployed automatically to stage.. Updating the PR in minutes to cover the suggestion.

Reply:
Changes are covered by 1899b54